Auth0/WebAuth.swift

```diff
@@ -162,8 +162,8 @@ public protocol WebAuth: Trackable, Loggable {
     /// - Returns: the same WebAuth instance to allow method chaining
     func responseType(_ response: [ResponseType]) -> Self
 
-    /// Add nonce paramater for authentication, this is a requirement for
-    /// when response type .id_token is specified.
+    /// Add `nonce` paramater for authentication, this is a requirement
```

Comment:
typo. parameter

Reply:
Fixed in ca28451be12118c8928745aead45feaa63e669e1.